packages/Avalonia/AvaloniaPrivateApis.targets

```diff
@@ -21,6 +21,6 @@
       <ReferencePath Include="$(MSBuildThisFileDirectory)/../lib/$(AvaloniaUnstableApiFrameworkToUse)/*.dll"/>
       <ReferencePathWithRefAssemblies Include="$(MSBuildThisFileDirectory)/../lib/$(AvaloniaUnstableApiFrameworkToUse)/*.dll"/>
     </ItemGroup>
-    <Warning Text="AvaloniaAccessUnstablePrivateApis is Enabled: This means you are using unstable internal APIs, and your code may be depending on APIs which may change or be removed in future versions of Avalonia. Set AvaloniaAccessUnstablePrivateApis to 'False' to disable this warning." />
+    <Warning Code="AVLN1002" Text="AvaloniaAccessUnstablePrivateApis is Enabled: This means you are using unstable internal APIs, and your code may be depending on APIs which may change or be removed in future versions of Avalonia. Set AvaloniaAccessUnstablePrivateApis to 'False' to disable this warning." />
```

Comment:
Two problems here:
1. AVLN prefix is used by the XAML compiler. We probably can change that to "AVX" as for "AValoniaXaml" before 11.1.
2. Specifically AVLN1000 range is used by the XML parsing errors.

Avalonia property analyzer uses AVP0000 prefix.
Avalonia property name generator uses AXN0000 prefix.

Reply:
There should be another non-conflicting code

Reply:
Hi, @maxkatz6 
I chose this prefix because I believe it was related to XAML compiler. What prefix do you suggest?

Reply:
I don't think Private APIs warning is related to the XAML compiler. You can avoid XAML completely and still get this warning.

Seems like we also have a bit more generic codes: AVA2001
https://github.com/AvaloniaUI/Avalonia/blob/73f7fd1d3eed0a9cb0bb7757a1ee1b728f68af6a/src/tools/Avalonia.Analyzers/OnPropertyChangedOverrideAnalyzer.cs#L13

Let's go with simple `AVA3001` for now.

Reply:
All of these codes should be documented later

Reply:
https://github.com/AvaloniaUI/avalonia-docs/issues/285